CONTRIBUTING.md

```diff
@@ -131,8 +128,8 @@ above, before submitting the PR to review:
 ```shell script
 BITCOIN_MSRV=1.29.0 ./contrib/ci.sh
 ```
-Where value in `BITCOIN_MSRV=1.29.0` should be replaced with the current MSRV
-from [README.md].
+Please replace the value in `BITCOIN_MSRV=1.29.0` with the current MSRV from 
```

Comment:
In 7ac1fc5f6473838988a55b9fc65d965d3079ff46:

More trailing spaces.

Reply:
How do you see them?
For me they are usually all get removed by `rustfmt`, but here we do not use that - and my eyes are not as laser as yours to see them clearly :D :D

Reply:
`grep ' $' README.md` should do the trick. :)

Reply:
It did, but my eyes does not run `grep` as an internal extension. I tried CRISPR for that, but grep is still missed from existing CRISPR repos for human genes :D

Reply:
I use [this tool as my git-diff extension](https://github.com/so-fancy/diff-so-fancy) which highlights trailing whitespace in red.

I am hesitant to recommend it because of the obvious security implications of replacing your diff tool, but it seems to have been working great for me (and has never shown me a diff inconsistent with the normal diff utility, in the times that I've checked).

Reply:
`git diff` shows trailing whitespace. You might need `--color=always` if you pass the output to a pager. FTR I use `vimpager` and have the following shell fuction along with a shell alias for it.
```
git-log-patch () {
    local commit
    if (( $# < 1 ))
    then
        commit="HEAD~"
    else
        commit="$1"
    fi
    git log --color=always --patch --reverse "${commit}".. | vimpager
}
```

Can be used like: `git-log-patch @~~` to get diffs of the last two patches.